cabal.project

```diff
@@ -12,6 +12,7 @@ packages:
    lib/amazonka-core/
    lib/amazonka-test/
    lib/amazonka-s3-encryption/
+   lib/amazonka-attribute-value/
```

Comment:
I think we should be clear that this is specific to DynamoDB; how about `amazonka-dynamodb-attributevalue`?
```suggestion
   lib/amazonka-dynamodb-attributevalue/
   lib/amazonka-s3-encryption/
```